### PR TITLE
feat!: `ProgressCircle` uses `new`-mode as default one

### PR DESCRIPTION
**Pull request by nsbarsukov**

### CODE REVIEW

File: projects/kit/components/progress/progress-circle/progress-circle.style.less [hunk 1]
```diff
@@ -11,94 +10,43 @@
...
 :host {
+    --t-track-stroke: var(--tui-thickness, 0.375em);
+    --t-progress-stroke: var(--tui-thickness, 0.375em);
+
     position: relative;
     display: block;
     color: var(--tui-primary);
     transform: rotate(-90deg);
     transform-origin: center;
-    font-size: 1rem; // to use em units inside svg (Safari doesn't support rem units inside svg)
```
Comment (nsbarsukov) [inline projects/kit/components/progress/progress-circle/progress-circle.style.less:44]: **TODO**: check Safari 13-17

Comment (nsbarsukov) [reply projects/kit/components/progress/progress-circle/progress-circle.style.less:44]: **iPhone 15 Pro Max IOS 17.3:**

Comment (nsbarsukov) [reply projects/kit/components/progress/progress-circle/progress-circle.style.less:44]: **iPhone 14 IOS 16.5** ✅

Comment (nsbarsukov) [reply projects/kit/components/progress/progress-circle/progress-circle.style.less:44]: **iPhone 13 IOS 15.4** ❌

Comment (nsbarsukov) [reply projects/kit/components/progress/progress-circle/progress-circle.style.less:44]: I need to revert `rem` => `em` + add `TODO`

Comment (nsbarsukov) [reply projects/kit/components/progress/progress-circle/progress-circle.style.less:44]: After this reversion https://github.com/taiga-family/taiga-ui/pull/7648/commits/3bbba0e143f6be1d00f5fa6a5fac7b13aff04634 (`rem` => `em`).

**iPhone 13 IOS 15.4** ✅

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**